consensus/fork/galactica.go

```diff
@@ -144,3 +145,15 @@ func GalacticaPriorityPrice(tr *tx.Transaction, baseGasPrice, provedWork *big.In
 	 */
 	return math.BigMin(feeItems.MaxPriorityFee, new(big.Int).Sub(feeItems.MaxFee, galacticaItems.BaseFee))
 }
+
+func CalculateReward(gasUsed uint64, rewardGasPrice, rewardRatio *big.Int, isGalactica bool) *big.Int {
+	reward := new(big.Int).SetUint64(gasUsed)
+	reward.Mul(reward, rewardGasPrice)
```

Comment:
Probably not, but triple checking, it's not required to `reward.Div(reward, big.NewInt(1e18))` here ?

Reply:
Correct, only when calculating the 30% of the reward (instead of ... * 0.3 it's ... * 3e17 / 1e18) when we are before galactica